packages/react-component-library/src/components/TopLevelNavigation/SidebarE/Sidebar.test.tsx

```diff
@@ -19,24 +24,22 @@ import {
 import { Link } from '../../Link'
 import { Notification, Notifications } from '../NotificationPanel'
 
-const userWithLink = (
-  <SidebarUserE
-    initials="HN"
-    name="Horatio Nelson"
-    link={<Link href="/user-profile">Profile</Link>}
-  />
-)
+const user = <SidebarUserE initials="HN" name="Horatio Nelson" />
 
 const userWithMenu = (
   <SidebarUserE
     initials="HN"
     name="Horatio Nelson"
-    link={<Link href="/user-profile">View profile</Link>}
+    userLink={<Link href="/user-profile">Profile</Link>}
```

Comment:
I am trying to understand why the consumer would compose the component with `userLink` and `exitLink` rather than just supplying everything as `children`.

Reply:
The children allow the sub-menu (only available on in the collapsed sidebar state) to be populated with optional arbitrary menu items. We need a way differentiating the user and exit links from these.

When the sidebar is in an expanded state the user and exit links are associated with an exit button and a 'View profile' link that has hardcoded text.

Maybe it doesn't make sense from a UX perspective to allow for the arbitrary children.

<img width="295" alt="Screenshot 2020-11-20 at 13 01 13" src="https://user-images.githubusercontent.com/48086589/99802961-808bc400-2b30-11eb-8066-1fe095877bb5.png">

<img width="161" alt="Screenshot 2020-11-20 at 13 00 53" src="https://user-images.githubusercontent.com/48086589/99802953-7d90d380-2b30-11eb-9bdb-51c2b64a205a.png">

Reply:
We have discussed this and decided to remove the optional arbitrary children for now.